.pipelines/e2e.yml

```diff
@@ -16,26 +20,43 @@ jobs:
 - job: E2E
   timeoutInMinutes: 180
   pool:
-    name: ARO-CI
-    demands: go-1.17
+    name: 1es-aro-ci-pool
   steps:
   - template: ./templates/template-checkout.yml
 
+  - script: |
+      set -xe
+      sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+      sudo dnf install -y openvpn make
```

Comment:
do we need sudo here? (since we are running as user 0)

Reply:
the Azure Pipelines stuff switches our user to the one that it creates after it sets up the container. The --user=0 is so that the initial setup doesn't run as user 1001 as set in the dockerfile.